Archive.Zip.UnitTests/ZipFileBuilderTest.cs

```diff
@@ -152,7 +152,7 @@ public void BuildReturnsZipFileWithEmptyFile_FromInMemoryFile ()
     [Test]
     public void BuildReturnsZipFileWithFileWithUmlaut ()
     {
-      string fileWithUmlautInName = Path.Combine (Path.GetTempPath(), "NameWith�.txt");
+      string fileWithUmlautInName = Path.Combine (Path.GetTempPath(), "NameWithÄ.txt");
```

Comment:
There's still a change here, line ending maybe?

Reply:
The changes, where it seems that nothing has been changed, are encoding fixes. It doesn't show it on github, but german umlauts were broken because of automatic encoding changes by Rider. I will resolve all related conversations to this.

Reply:
The file encoding changed from `ANSI` to `UTF-8`. I don't think this is needed, as `ANSI` supports umlauts, the file was fine before the changes after all.

Reply:
Same Problem exists during the NUnit3 migration and the files end up as Unicode. When we touch the file and update the encoding, that's okay. Just need to make sure that the umlauts get patched accordingly, too.